recipes/pfr/all/conandata.yml

```diff
@@ -1,7 +1,16 @@
 sources:
   "1.0.4":
-    - url: "https://github.com/boostorg/pfr/archive/1.0.4.tar.gz"
+    - url: "https://github.com/boostorg/pfr/archive/refs/tags/1.0.4.tar.gz"
       sha256: "01ecb27c850f50c589bad9a741ec9af6b06fe07ed673946c4bbb1960eb0f126e"
+  "2.0.0":
```

Comment:
Do you require these older versions? We like to suggest removing them if not :)

https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-for-adding-older-versions-of-a-package

Reply:
Sure, that makes sense.